elliott

```diff
@@ -657,11 +657,8 @@ PRESENT advisory. Here are some examples:
     $ elliott --group openshift-3.6 find-builds -k rpm -b megafrobber-1.0.1-2.el7 -b 93170
 """
 
-    if (from_diff and kind) or (from_diff and builds) or (builds and kind):
-        raise ElliottFatalError("Use only one of --kind or --build or --from-diff.")
-
-    if not builds and not from_diff and not kind:
-        raise ElliottFatalError("Must use one of --kind or --build or --from-diff.")
```

Comment:
I think we still want this check for the original spirit of the reverted fix. We do need to specify one of these or we get a confusing error.

Additionally, IIRC if we specify `--build` we still need `--kind` or the code gets confused. So a condition to ensure that as well?